Schronisko_app.py

```diff
@@ -0,0 +1,129 @@
+from flask import Flask, request, render_template, g, flash, redirect, url_for, abort
+
+import sqlite3
+
+DATABASE = '/path/to/database.db'
+app = Flask(__name__)
+app.secret_key = "TjWnZq4t7w!z%C*F"
+app_info = {
+    "db_file": "database/schronisko.db"
+}
+"""
+create table animals(id integer primary key autoincrement, name varchar(35), category varchar (40), race varchar(50), 
+description text, admisson_date date not null default(date()));
+"""
```

Comment:
Can you please automate this? Script would check if this table exists and create it if needed